sdcm/sla/libs/sla_utils.py

```diff
@@ -12,6 +14,14 @@ class SchedulerRuntimeUnexpectedValue(Exception):
     pass
 
 
+class SchedulerGroupNotFound(Exception):
+    pass
+
+
+class WrongServiceLevelShares(Exception):
+    pass
+
+
```

Comment:
recently we added `sdcm.exceptions` where we put them

Reply:
since those are specific to SLA, maybe it's o.k. to leave them here (at least until there a reason to seperate)